frontend/src/Login/Signup.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import {Link} from 'react-router-dom'
+
+export default function Signup() {
+  return (
+    <div className='logincontainer'>
+        <div className='welcomecontainer'>
+          <h1>Welcome!</h1>
+          <p>Commited to serivce excellence</p>
+          <div className='socialmediaicons'>
+              <i className="fa-brands fa-facebook"></i>
+              <i className="fa-brands fa-twitter"></i>
+              <i className="fa-brands fa-instagram"></i>
+              <i className="fa-solid fa-envelope"></i>
+          </div>
+        </div>
+
+        <div className='authenticationcontainer'>
+          <h2>Sign Up</h2>
+          <form>
+
+            <div className='entry'>
+              <label htmlFor="Username">Username</label>
+              <input type="text" />
+            </div>
+
+            <div className='entry'>
+              <label htmlFor="Email Address">Email Address</label>
+              <input type="email" />
+            </div>
+
+            <div className='entry'>
+              <label htmlFor="Password">Create Password</label>
+              <input type="password" />
+            </div>
+
+            <div className='role'>
```

Comment:
Instead of "Role", can we call it "Select role" to denote a required action?
Also,  in your screenshot, I'm actually not too sure where this is rendered (I just see the dropdown options but no input component). What do you click on the render the dropdown?

Reply:
@basticks02 if you're adding/updating a flow in your application, it might be worth adding a screen recording as opposed to the screenshot

Reply:
Okay.